src/compose/volume.ts

```diff
@@ -1,16 +1,15 @@
 import * as Docker from 'dockerode';
-import assign = require('lodash/assign');
-import isEqual = require('lodash/isEqual');
-import omitBy = require('lodash/omitBy');
-
-import constants = require('../lib/constants');
+import * as _ from 'lodash';
+import * as constants from '../lib/constants';
 import { docker } from '../lib/docker-utils';
 import { InternalInconsistencyError } from '../lib/errors';
 import * as LogTypes from '../lib/log-types';
 import { LabelObject } from '../lib/types';
 import * as logger from '../logger';
 import * as ComposeUtils from './utils';
 
+const { assign, isEqual, omitBy } = _;
```

Comment:
If you're going to import all of lodash/destructure it'd be much better to do
```suggestion
import { assign, isEqual, omitBy } from 'lodash';
```

Reply:
I tried to do that, but doesn't work without enabling `esModuleInterop`. Glad you mention it though, can you think of any drawbacks to enabling that option on tsconfig?

Reply:
I don't think you need to enable `esModuleInterop` for it to work - I've seen it elsewhere without needing to enable `esModuleInterop`. Maybe you need to update lodash typings?

Reply:
Oh, you are right, I think I got it confused with the `import assign from 'lodash/assign'` and its variants that do require esModuleInterop